lib/rules/no-multi-spaces.js

```diff
@@ -99,3 +99,21 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.schema = [
+    {
+        "type": "object",
+        "properties": {
+            "exceptions": {
+                "type": "object",
+                "patternProperties": {
+                    "^([A-Z][a-z]*)+$": {
```

Comment:
I suppose the extra restriction can't hurt. I just pushed a commit to change it back.